qmail-local.c

```diff
@@ -42,6 +43,20 @@ void temp_slowlock()
 void temp_qmail(fn) char *fn;
 { strerr_die5x(111,"Unable to open ",fn,": ",error_str(errno),". (#4.3.0)"); }
 
+/* writes ulong u in hex to char *s, does not NULL-terminate */
+unsigned int fmt_xlong(s,u) char *s; unsigned long u;
+{
+ unsigned int len; unsigned long q; unsigned long c;
```

Comment:
Please unwrap them also, which also gives the opportunity to initialize them immediately and get rid of the next line entirely.

Reply:
fmt_xlong() was removed from qmail-local.c a while back and this regression will be fixed in a new push.